scripts/run_tests.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3
+
+import os
+import subprocess
+import sys
+from pathlib import Path
+
+
+def load_env_file():
```

Comment:
We do not need this complicated logic. Simply load with `load_dotenv()` and proceed. These scripts expect the `.env` to be set up.